### PR TITLE
doc: add minutes for CTC meeting 29 March 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-03-29.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+## Minutes
+
+### buffer: auto random fill Buffer(num) and new Buffer(num) [#11808](https://github.com/nodejs/node/pull/11808) and buffer: discuss future direction of Buffer constructor API [#9531](https://github.com/nodejs/node/issues/9531)
+
+* Rich started by framing the current state of the issue. Some
+  consensus that we should do something.  Had looked like 
+  option to do opt-in deprecate flag was non-controversial, but
+  seems like there is some questioning/objections.  
+* We may need to start with if we are going to state that it 
+  will be deprecated at some point as that affects people.s
```
Comment (bnoordhuis) [inline meetings/2017-03-29.md]: s/\.s/'s/ -- happens elsewhere in the document too (also s/\.t/'t/.)

File: meetings/2017-03-29.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    **Zero**: @jasnell, @fishrock123, @ofrobots, @trevnorris, @mhdawson,
+      @targos, @addaleax, @evanlucas, @cjihrig
+      
+    **Random**: @addaleax, @cjihrig, @targos, @fhinkel
+    
+    **Neither**: @mscdex, @chalker
+    
+    **Abstain**: 
+
+  * Vote 4: If we chose to fill, do we backport (semver-minor)
```
Comment (bnoordhuis) [inline meetings/2017-03-29.md]: s/chose/choose/

**Review by bnoordhuis — APPROVED** [review]

File: meetings/2017-03-29.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+  consensus that we should do something.  Had looked like 
+  option to do opt-in deprecate flag was non-controversial, but
+  seems like there is some questioning/objections.  
+* We may need to start with if we are going to state that it 
+  will be deprecated at some point as that affects people.s
+  decisions on the other issues.
+* James: we need 3 distinct votes
+  * commit to runtime deprecation and timeframe
+  * do we put in pending deprecation
+  * do we random fill or zero fill by default, and do we backport
```
Comment (sam-github) [inline meetings/2017-03-29.md:97]: zero fill, random fill leads to random bugs that are hard to reproduce.

File: meetings/2017-03-29.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+    **NO** (Do not **commit** to deprecating): @evanlucas, @addaleax, @jasnell,
+      @ofrobots, @Fishrock123, @mhdawson, @cjihrig
+
+    **ABSTAIN**: @trott, @trevnorris
+
+    Need to take back to github, to get additional CTC members to vote.
+
+  * Vote2: Do we do a conditional pending deprecation warning.
+    Normal deprecation warning which will only be seen if if they opt in with
```
Comment (sam-github) [inline meetings/2017-03-29.md]: "if if"

File: meetings/2017-03-29.md (hunk 3 above)
Comment (seishun) [reply meetings/2017-03-29.md:97]: @sam-github ~~That's an interesting point I hadn't thought about before. Would you mind mentioning it in https://github.com/nodejs/node/pull/11808?~~ Actually no. It wouldn't make it any worse than it is now in that regard.

**Comment by mhdawson** [comment]
@bnoordhuis @sam-github, pushed updates to address comments.

**Review by evanlucas — APPROVED** [review]

**Review by bnoordhuis — APPROVED** [review]

**Review by joshgav — APPROVED** [review]

**Review by jasnell — APPROVED** [review]

**Pull request merged**